business-registry-dashboard/app/components/Form/AddBusiness/Base.vue

```diff
@@ -170,7 +170,9 @@ async function submitManageRequest () {
   }
 
   loading.value = true
-  // await handleRemoveExistingAffiliationInvitation() // TODO: figure out if this is necessary, I do not think it is
+
+  await affStore.deletePendingInvitations(props.identifier) // delete any existing pending affiliations
```

Comment:
delete any pending invitations for the current business, this is necessary because it will error if there's an existing invitation when submitting a new request